cmd/terramate/cli/script_list.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package cli
+
+import (
+	"sort"
+
+	"github.com/terramate-io/terramate/config"
+	"github.com/terramate-io/terramate/hcl"
+	project2 "github.com/terramate-io/terramate/project"
+)
+
+type scriptListEntry struct {
+	ScriptCfg *hcl.Script
+	Dir       string
+	DefCount  int
+}
+
+type scriptListMap map[string]*scriptListEntry
+
+func (c *cli) printScriptList() {
+	srcpath := project2.PrjAbsPath(c.rootdir(), c.wd())
```

Comment:
this same directory imports "project" as "prj". Can you change here to use the same for now?

We can fix this in a separate PR renaming the "project" type so this conflict does not happen again.